src/portuguese.js

```diff
@@ -0,0 +1,43 @@
+
+class PresenteIndicativo {
+  conjugate (word, info) {
+    let ending = [];
+    switch (word.slice(-2)) {
+      case 'ar':
+        ending = info.singular ? ['o', 'as', 'a'] : ending = ['amos', 'ais', 'am'];
+        break;
+      case 'er':
+        ending = info.singular ? ['o', 'es', 'e'] : ['emos', 'eis', 'em'];
```

Comment:
if we use subject, we could eliminate singular and person, then we'd only need _mood_, _tense_ and _subject_.